ios/BT/Storage/KeychainService.swift

```diff
@@ -0,0 +1,25 @@
+import KeychainAccess
+
+final class KeychainService {
+
+  static let shared = KeychainService()
+
+  private lazy var keychain = Keychain(service: "\(Bundle.main.bundleIdentifier!).keychainService")
```

Comment:
I see, so we will be storing the token on a combination of bundle identifier and then dev and prod environments. Sounds good but it seems to me that we will still have the problem if we change the server URL on the environment variables while we are at the dev environment we will run into the same issue. Is it possible to store the key using a combination of the bundle identifier and the exposures server? Could we store the tokens in some sort of dictionary where the keys are the exposure server URL and that could be persisted per install? Again not bulletproof but will mimic the behavior that is being problematic to us which is using the same keys, on the same server with the wrong revision token.